### PR TITLE
docs:added env

### PR DESCRIPTION
**Pull request by ashishkava**

### CODE REVIEW

**Review by JainishaParmar — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGO_URL = mongodb://localhost:27017/Tutorials
```
Comment (kapilbarad) [inline .env:1]: Please take the ref - https://github.com/jaimin45/demo-nodejs-tutorials/blob/feature/envs/README.md

**Review by kapilbarad — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+## Install the dependencies and devDependencies and start the server.
+
+```sh
+npm install
+```
+## Load env vars
+
+```sh
+MONGO_URL="mongodb://localhost:27017/Tutorials"
+cp.env
```
Comment (kapilbarad) [inline README.md:35]: cp.env isn't required.
Also please list other envs here, like PORT, LOG_LEVEL

**Review by kapilbarad — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ npm install
 
 ```sh
 MONGO_URL="mongodb://localhost:27017/Tutorials"
+PORT = 8080
+LOG_LEVEL
```
Comment (kapilbarad) [inline README.md:36]: You need to set an example value here: LOG_LEVEL=debug

**Review by kapilbarad — CHANGES_REQUESTED** [review]

File: .env [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (kapilbarad) [inline .env:1]: We should not commit the .env file in github. Please remove this and add it in .gitignore file.

File: init/db.js [hunk 5]
```diff
@@ -1,8 +1,8 @@
+require('dotenv').config();
```
Comment (kapilbarad) [inline init/db.js:1]: This should be in www.js file.

**Review by kapilbarad — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -29,12 +29,10 @@
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
     "dotenv": "^10.0.0",
```
Comment (kapilbarad) [inline package.json:31]: `dotenv`  Should be in Dev Dependencies.

**Review by kapilbarad — APPROVED** [review]

**Pull request merged**